assembl/tests/fixtures/ideas.py

```diff
@@ -95,9 +96,9 @@ def fin():
 
 
 @pytest.fixture(scope="function")
-def subidea_1(request, discussion, root_idea, test_session, ):
-    """An Idea fixture with a idealink to root idea fixture -
-    root_idea
+def subidea_1(request, discussion, root_idea, test_session):
+    """A subidea fixture with a idealink to root idea fixture -
+    top_idea
```

Comment:
Why did you change root_idea to top_idea here?

Reply:
While discussing with @ayyazdaniaryan what I understood is that there is just one root idea in a discussion, from which goes out top ideas and from each each top idea goes subideas. In the code comments the terminology is a bit different ... the root idea is actually treated as a top idea ... What do you think should I revert it ?

Reply:
root_idea is really root_idea here, it's discussion.root_idea. subidea_1 is actually your top idea, level 1. Just revert your docstring change here.